libraries/AC_AttitudeControl/AC_PosControl.cpp

```diff
@@ -84,238 +84,243 @@ const AP_Param::GroupInfo AC_PosControl::var_info[] = {
+    // @Param: _D_VEL_IMAX
+    // @DisplayName: Velocity (down) controller I gain maximum
+    // @Description: Velocity (down) controller I gain maximum.  Constrains the target acceleration that the I gain will output
     // @Range: 1.000 8.000
     // @User: Standard
 
-    // @Param: _VELZ_D
-    // @DisplayName: Velocity (vertical) controller D gain
-    // @Description: Velocity (vertical) controller D gain.  Corrects short-term changes in velocity
+    // @Param: _D_VEL_D
+    // @DisplayName: Velocity (down) controller D gain
+    // @Description: Velocity (down) controller D gain.  Corrects short-term changes in velocity
     // @Range: 0.00 1.00
     // @Increment: 0.001
     // @User: Advanced
 
-    // @Param: _VELZ_FF
-    // @DisplayName: Velocity (vertical) controller Feed Forward gain
-    // @Description: Velocity (vertical) controller Feed Forward gain.  Produces an output that is proportional to the magnitude of the target
+    // @Param: _D_VEL_FF
+    // @DisplayName: Velocity (down) controller Feed Forward gain
+    // @Description: Velocity (down) controller Feed Forward gain.  Produces an output that is proportional to the magnitude of the target
     // @Range: 0 1
     // @Increment: 0.01
     // @User: Advanced
 
-    // @Param: _VELZ_FLTE
-    // @DisplayName: Velocity (vertical) error filter
-    // @Description: Velocity (vertical) error filter.  This filter (in Hz) is applied to the input for P and I terms
+    // @Param: _D_VEL_FLTE
+    // @DisplayName: Velocity (down) error filter
+    // @Description: Velocity (down) error filter.  This filter (in Hz) is applied to the input for P and I terms
     // @Range: 0 100
     // @Units: Hz
     // @User: Advanced
 
-    // @Param: _VELZ_FLTD
-    // @DisplayName: Velocity (vertical) input filter for D term
-    // @Description: Velocity (vertical) input filter for D term.  This filter (in Hz) is applied to the input for D terms
+    // @Param: _D_VEL_FLTD
+    // @DisplayName: Velocity (down) input filter for D term
+    // @Description: Velocity (down) input filter for D term.  This filter (in Hz) is applied to the input for D terms
     // @Range: 0 100
     // @Units: Hz
     // @User: Advanced
-    AP_SUBGROUPINFO(_pid_vel_u_cm, "_VELZ_", 3, AC_PosControl, AC_PID_Basic),
+    AP_SUBGROUPINFO(_pid_vel_d_m, "_D_VEL_", 12, AC_PosControl, AC_PID_Basic),
 
-    // @Param: _ACCZ_P
-    // @DisplayName: Acceleration (vertical) controller P gain
-    // @Description: Acceleration (vertical) controller P gain.  Converts the difference between desired vertical acceleration and actual acceleration into a motor output
+    // @Param: _D_ACC_P
+    // @DisplayName: Acceleration (down) controller P gain
+    // @Description: Acceleration (down) controller P gain.  Converts the difference between desired vertical acceleration and actual acceleration into a motor output
```

Comment:
if we're dividing by 10 then should also change "@Range" should change here and in other params
need to check all changed params

Reply:
it would be good to test how bad it is with the old 10x params

Reply:
can we get unit annotations on the param docs for these changed params?